### DISCUSSION TITLE
How to get enum values as strings

### DISCUSSION THREAD

**Discussion by moroshko** [prisma/prisma#7451] [Q&A]
Say that `schema.prisma` has the following `enum` definition:
```prisma
enum Color {
  RED
  GREEN
  BLUE
}
```

I'd like to render a `<select>` with these 3 values.

How could I generate an array/object with these values as strings?

```ts
["RED", "GREEN", "BLUE"]

// or

{
  "RED": "RED",
  "GREEN": "GREEN",
  "BLUE": "BLUE"
}
```

**Comment by ryands17** [comment]
@moroshko 👋 
Currently the only way to do this would be to specifically write all your enums in an array as follows:

```ts
import { Color } from '@prisma/client'

const colors = [Color.RED, Color.GREEN, Color.BLUE]
```

This is due to enums generated being string enums and not regular enums. It would be great if you could add a 👍 to this request and follow it for more updates.
https://github.com/prisma/prisma/issues/5609

**Reply by moroshko** [reply]
Thanks @ryands17! 

`Object.keys(Color)` gives me the array I wanted.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
